extensions/VisualEditor/wikia/modules/ve/ui/dialogs/ve.ui.WikiaMediaInsertDialog.js

```diff
@@ -600,7 +613,7 @@ ve.ui.WikiaMediaInsertDialog.prototype.insertPermanentMedia = function ( cartIte
  * @param {Object} items Items to insert
  * @param {ve.dm.SurfaceFragment} fragment
  */
-ve.ui.WikiaMediaInsertDialog.prototype.insertPermanentMediaCallback = function ( items, fragment ) {
+ve.ui.WikiaMediaInsertDialog.prototype.insertPermanentMediaCallbackWrapper = function ( items, fragment, label ) {
```

Comment:
Why not just create a property such as ```trackingPrefix``` and have if very between different classes? Very common pattern in VE - and proven to works nice at scale.